db/migrate/20210607105903_add_admin_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddAdminToUser < ActiveRecord::Migration[6.1]
+  def change
+    add_column :users, :admin, :boolean
```

Comment:
WDYT :arrow_down: ?
```suggestion
    add_column :users, :admin, :boolean, null: false, default: false
```

Reply:
Doesn't matter, because if User is not admin: true, it is not employee and can't create cinema_halls, movies, etc. It can be nil. Employee is getting rank by updating through console.

Reply:
Notice that `admin` could be `true`, `false` or `nil` in the current shape.